pkg/controller/storagecluster/cephobjectstores.go

```diff
@@ -28,7 +28,7 @@ func (r *ReconcileStorageCluster) ensureCephObjectStores(instance *ocsv1.Storage
 
 	var cephObjectStores []*cephv1.CephObjectStore
 	if cephObjectStores, err = r.newCephObjectStoreInstances(instance); err == nil {
-		err = r.createCephObjectStores(cephObjectStores, instance, reqLogger)
+		err = r.createCephObjectStores(cephObjectStores, reqLogger)
```

Comment:
Why is this being removed here? Was it never used?

Reply:
It was never used in the `createCephObjectStores()` function.